internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,63 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+
+	output := strings.Split(data, ",")
+
+	if len(output) != 2 {
+		return 0, 0, fmt.Errorf("некорректное количество параметров")
+	}
+
+	steps, err := strconv.Atoi(output[0])
+
+	if err != nil {
+		return 0, 0, err
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```